content/docs/state-and-lifecycle.md

```diff
@@ -155,9 +156,9 @@ Note how we pass `props` to the base constructor:
   }
 ```
 
-Class components should always call the base constructor with `props`.
+क्लास के components को हमेशा बेस कंस्ट्रक्टर को `props` के साथ कॉल करना चाहिए।
 
-3) Remove the `date` prop from the `<Clock />` element:
+3) `<Clock />` एलिमेंट से `date` prop निकालें:
```

Comment:
`date` prop ko kuch aise nikale

Reply:
Seems right the way it is. As "ko kuch aise nikale" is long and does not fit right.